### ISSUE TITLE
Support ESLint 9.x

### ISSUE THREAD

**Issue by rhwood** [RedHatInsights/frontend-components#2124]
Since ESLint [8.x is EOL](https://eslint.org/version-support/) can ESLint 9.x be supported?

**Comment by Hyperkid123** [comment]
Yes. We have recently made some updates to the repo management tooling and this will follow.

We have a ticket in the backlog: https://issues.redhat.com/browse/RHCLOUD-32095. If you this is a blocker for you or need some feature from eslint 9, please add it them to the ticket.
